Header header
#std_msgs/Int32MultiArray tsrdim
#std_msgs/Float32MultiArray[] tsrlist
geometry_msgs/PoseArray[] poses